### DISCUSSION TITLE
Adding more buttons?

### DISCUSSION THREAD

**Discussion by EcoVeni** [KadeArchive/Kade-Engine#1257] [Q&A] [answered]
Does anybody know how to add more buttons on the menu screen?

**Comment by isakube** [comment, marked as answer]
it requires more fine-tuning but you can just add stuff to `optionShit` in MainMenuState. it wont load graphics unless you actually get them set up (FNF_main_menu_assets.png / .xml)

to make them do stuff just find `if (controls.ACCEPT)` in the same file and do stuff there
